web/src/engine/websites/NetTruyenOK.ts

```diff
@@ -4,15 +4,17 @@ import { DecoratableMangaScraper } from '../providers/MangaPlugin';
 import * as MojoPortalComic from './templates/MojoPortalComic';
 import * as Common from './decorators/Common';
 
+const pageScript = `[...document.querySelectorAll('div.page-chapter > img:not([src^="data:image"]):not([src*="imgur.com"])')].map(image => image.src);`;
+
 @Common.MangaCSS(new RegExp(`^{origin}/manga/[^/]+$`), MojoPortalComic.queryManga)
 @Common.MangasMultiPageCSS('/danh-sach-truyen/{page}', MojoPortalComic.queryMangas)
 @Common.ChaptersSinglePageCSS(MojoPortalComic.queryChapters)
-@Common.PagesSinglePageCSS('div.page-chapter > img:not([src^="data:image"]):not([src*="imgur.com"])')
+@Common.PagesSinglePageJS(pageScript, 1500)
```

Comment:
Why using a script? The [raw HTML](https://nettruyendex.com/manga/ta-co-90-ty-tien-liem-cau/chapter-200) content seems just fine to be extracted with CSS selector

Reply:
That is totally random. I swear that i have seen lazyloaded images